### PR TITLE
hyperparameter tuning for td learning, solves all puzzles except 6,8,…

### PR DESCRIPTION
**Pull request by jihyopark02**
can't solve 6,8,9,10

9 worked before, but not working now for some reason.

changed gamma, alpha (learning rate) and early stopping patience. read comments in the file. didn't touch constants.py

### CODE REVIEW

**Comment by omartounsi7** [comment]
We should make a new pr, for hyperparameter tuning of all algorithms, with the new code on main.
